Start.bat

```diff
@@ -1,16 +1,27 @@
 cd %~dp0
 
+setlocal enabledelayedexpansion
+set wallet=1Q24z7gHPDbedkaWDTFqhMF8g7iHMehsCb
+set username=aaronsace
+set workername=multipoolminer
+set region=europe
+set currency=btc,usd,eur
+set type=amd,nvidia,cpu
+set poolname=miningpoolhub,miningpoolhubcoins,zpool,nicehash
+set algorithm=equihash,skunk,decred,sia,lbry,pascal,ethash,siaclaymore,keccak,lyra2re2,hsr,phi,jha,timetravel,tribus,x11evo,polytimos,x17,c11,skein,sib,nist5,myr-gr,blakecoin,blake2s,veltor,cryptonight,groestl,lyra2z,neoscrypt,yescrypt,xevan,vanilla,keccakc,jha,blakevanilla,blakevanilla,ethash2gb,hmq1725
```

Comment:
This does not match the original list.

Reply:
I'd rather keep all that data in a separate file so it can be used by all batch files.
Ideally we create a GUI to create above config file - even a compiled ps1 file with GUI elements could do.